package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "@tensor-hq/tensor-common",
-  "version": "5.0.0",
+  "version": "5.0.1",
```

Comment:
Nit: new functionality should be a minor version bump https://semver.org/

Reply:
done, @richardwu can you merge

Reply:
wait i'm codeowner now nvm :D